packages/salesforcedx-vscode-core/src/telemetry/telemetryMessages.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { commands, Uri, window } from 'vscode';
+import { TELEMETRY_OPT_OUT_LINK } from '../constants';
+import { nls } from '../messages';
+
+export const internalTelemetryMessage = async () => {
+    const internalMessage = nls.localize('telemetry_internal_user_message');
+    await window.showInformationMessage(internalMessage);
+};
```

Comment:
I was seeing the notification show up more then I expected it to.  So it looks like when you await this it will wait until the notification is dismissed to resolve the promise.  What do you think about making this 
`void window.showInformationMessage(internalMessage);`
That way the user doesn't have to dismiss the notification in order to not see it again.